### ISSUE TITLE
Does not copy to clipboard

### ISSUE THREAD

**Issue by ahonnecke** [mufeedvh/code2prompt#10]
```ahonnecke@antonym:~/src/wayward/src/wayward$ code2prompt .
▹▹▹▹▸ Done!                                                                                                                                                               [✓] Prompt copied to clipboard!
```

However, nothing had been copied to the clipboard.  Has this been tested on a linux machine?

**Comment by ahonnecke** [comment]
I can reproduce this when I build it locally as well.

**Comment by mufeedvh** [comment]
Hey @ahonnecke, thank you for opening an issue on this. Can you please send the complete output from code2prompt with `--tokens` flag used?

**Comment by rohan-paul** [comment]
Yes having same issue in Ubuntu 20.04 as well.

However absolutely great tool, and Thank You @mufeedvh  for open-sourcing it.

**Comment by RastislavKish** [comment]
Hello,

@mufeedvh the problem with Linux clipboard is that the running process is always responsible for providing the copied data, when the process exits, the information is efficiently lost (it's the same as if the clipboard was cleared out). There are workarounds for this problem, see for example the arboard examples, but it is sort of a drag to implement.

**Comment by ODAncona** [comment]
@RastislavKish Hello, it's fixed in #20 using the `use arboard::Clipboard;`

**Comment by mufeedvh** [comment]
Fixed in #20 by @ODAncona! 🙌 🎉
